rules/aip0233/plural_method_name_test.go

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0233
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestMethodPluralResourceName(t *testing.T) {
+	// Set up the testing permutations.
+	tests := []struct {
+		testName string
+		src      string
+		problems testutils.Problems
+	}{
+		{
+			testName: "Valid-BatchCreateBooks",
+			src: `import "google/api/annotations.proto";
```

Comment:
Consider to use a template to reduce boilerplate.

Reply:
Done.